tests/unit/AnnualReport.spec.ts

```diff
@@ -871,11 +871,12 @@ describe('Annual Report - Part 3 - Submitting', () => {
     sinon.restore()
   })
 
-  it('saves a new filing and redirects to Pay URL when this is a new AR and the File & Pay button ' +
+  xit('saves a new filing and redirects to Pay URL when this is a new AR and the File & Pay button ' +
```

Comment:
This needs to be fixed.

Reply:
Yes. You're going to see this in a couple of different places also. They're all the same error that I mentioned above. Once I fix one, it'll fix all of them.

Reply:
Two other comments already mention a new ticket for this, so I'm resolving this conversation.